README.md

```diff
@@ -133,6 +149,24 @@ f = f2 >>= f1
 
 </details>
 
+<details><summary>Function as Semigroup</summary>
+
+```haskell
+f x = f1 x <> f2 x
+f = f1 <> f2
+```
+
+</details>
+
+<details><summary>Function as Monoid</summary>
+
+```haskell
+f x = f1 x <> mempty
+f = f1 <> mempty
```

Comment:
Not sure if it's useful for composition :). 
But it is indeed a good brain exercise.

Reply:
@shineli1984 Yeah I wonder if there's any real life examples of using this one?